patches/minecraft/net/minecraft/world/storage/WorldInfo.java.patch

```diff
@@ -41,15 +75,27 @@
          }
  
          p_76064_1_.func_74782_a("DimensionData", nbttagcompound1);
-@@ -708,6 +711,7 @@
+@@ -387,6 +399,11 @@
+         {
+             p_76064_1_.func_74782_a("Player", p_76064_2_);
+         }
++
++        if (this.capabilities != null)
++        {
++            p_76064_1_.func_74782_a("capabilities", capabilities.serializeNBT());
```

Comment:
TileEntity, ItemStack and Entity capabilities (and upcoming Chunk capabilities) use the key `"ForgeCaps"`. World seems to be the odd one out with `"capabilities"`. You should probably stick to `"ForgeCaps"` for consistency.

Reply:
Oh wait, I forgot it completely. Thanks!